filetracker/scripts/recover.py

```diff
@@ -0,0 +1,130 @@
+"""Script for recovering filetracker storage consistency after failures.
+
+This script iterates over all existing links, removing broken ones and
+recalculating blob reference count from scratch, overwriting existing
+values in DB.
+
+It also iterates over blobs and removes blobs that are not linked.
+
+WARNING: this script does not use or respect locks, so DO NOT run
+this while storage is being used by a filetracker server.
+"""
+
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import argparse
+import os
+import sys
+
+import six
+
+from filetracker.scripts import progress_bar
+from filetracker.servers.storage import FileStorage
+
+# Value used for aligning printed action names
+_ACTION_LENGTH = 25
+
+
+def main():
+    parser = argparse.ArgumentParser(
+            description='Restores storage consistency after failures')
+    parser.add_argument('root', help='root directory of filetracker storage')
+    parser.add_argument('-s', '--silent', action='store_true',
+            help='if set, progress bar is not printed')
+
+    args = parser.parse_args()
+    root, silent = args.root, args.silent
```

Comment:
I don't see why these 2 assignments should be done in one line?
This suggests that they are somehow related, which I don't think is the case.

Reply:
Well, I guess this comes to subjective preference.
I'm just used to seeing such a construct either when unpacking a tuple that is a result of some function or in the trick to swap to variables. Seeing this construct suggests to me that something more intricate is going on, whereas here nothing very special happens, so that may be misleading.
However it also makes perfect sense, that as you've described the assignments are in fact related.
For me, the simpler version is less confusing, but I'm not opposed to the current version.